Makefile

```diff
@@ -15,13 +15,13 @@ MAKEFLAGS+=--no-builtin-rules
 INSTANCE_PORT=8080
 DOCKER_IMAGE=plone/server-dev:6.0.8
 DOCKER_IMAGE_ACCEPTANCE=plone/server-acceptance:6.0.8
-KGS=
+KGS=plone.restapi==9.2.0
 NODEBIN = ./node_modules/.bin
 SCRIPTSPACKAGE = ./packages/scripts
 
 # Plone 5 legacy
 DOCKER_IMAGE5=plone/plone-backend:5.2.12
-KGS5=plone.restapi==8.43.3 plone.volto==4.1.0 plone.rest==3.0.1
+KGS5=plone.restapi==9.2.0 plone.volto==4.1.0 plone.rest==3.0.1
```

Comment:
@sneridagh @davisagli from what I could find out, this  seems not to be sufficient to bump restapi up to 9.2.0 in the test backend. At least in my local test runs on this branch it stayed at 9.1.2

Reply:
I think we need a new release of https://hub.docker.com/r/plone/server-acceptance/tags here with the appropriate restapi version

Reply:
mmm I see, that is a problem :/ 
The acceptance images right now do not "compile" and are not `ADDONS` env aware... It would be great that they would be... For now, we would have to wait until 6.0.9, and have enough with FZJ ones.